content/docs/tasks/security/authn-policy/index.md

```diff
@@ -396,49 +408,78 @@ spec:
 EOF
 {{< /text >}}
 
-Re-run the request from `sleep.legacy`, we should see a success return code again (200), confirming service-level policy overrules the namespace-level policy.
+Re-run the request from `sleep.legacy`, we should see a success return code again (200), confirming service-specific policy overrules the namespace-wide policy.
 
 {{< text bash >}}
 $ kubectl exec $(kubectl get pod -l app=sleep -n legacy -o jsonpath={.items..metadata.name}) -c sleep -n legacy -- curl http://httpbin.foo:8000/ip -s -o /dev/null -w "%{http_code}\n"
 200
 {{< /text >}}
 
-## Setup end-user authentication
+### Cleanup part 2
+
+Remove policies and destination rules created in this session:
+
+{{< text bash >}}
+$ kubectl delete policy default overwrite-example -n foo
+$ kubectl delete policy httpbin -n bar
+$ kubectl delete destinationrules default overwrite-example -n foo
+$ kubectl delete destinationrules httpbin -n bar
+{{< /text >}}
+
+## End-user authentication
+
+You will need a valid JWT (corresponding to the JWKS endpoint you want to use for the demo). In this tutorial, we will use a test JWT signed by Google service account, but of course, you can also use your own JWT/JWKS endpoint for the demo.
 
-You will need a valid JWT (corresponding to the JWKS endpoint you want to use for the demo). Please follow the instructions [here]({{< github_tree >}}/security/tools/jwt) to create one. You can also use your own JWT/JWKS endpoint for the demo. Once you have that, export to some environment variables.
+You can create a test JWT for your Google service account (assume it's stored in the environment variable `$SVC_ACCOUNT`) by following the instructions [here]({{< github_tree >}}/security/tools/jwt). Keep it in `$TOKEN`. Also set `$JWKS` corresponding to the service account `$SVC_ACCOUNT` as below:
 
 {{< text bash >}}
-$ export SVC_ACCOUNT="example@my-project.iam.gserviceaccount.com"
-$ export JWKS=https://www.googleapis.com/service_accounts/v1/jwk/${SVC_ACCOUNT}
-$ export TOKEN=<YOUR-TOKEN>
+$ JWKS=https://www.googleapis.com/service_accounts/v1/jwk/${SVC_ACCOUNT}
 {{< /text >}}
 
-Also, for convenience, expose `httpbin.foo` via ingress (for more details, see [ingress task](/docs/tasks/traffic-management/ingress/)).
+Also, for convenience, expose `httpbin.foo` via ingressgateway (for more details, see [ingress task](/docs/tasks/traffic-management/ingress/)).
 
 {{< text bash >}}
-$ cat <<EOF | kubectl apply -f -
-apiVersion: extensions/v1beta1
-kind: Ingress
+$ cat <<EOF | istioctl create -f -
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: httpbin-gateway
+  namespace: foo
+spec:
+  selector:
+    istio: ingressgateway # use Istio default gateway implementation
+  servers:
+  - port:
+      number: 80
+      name: http
+      protocol: HTTP
+    hosts:
+    - "*"
+EOF
+$ cat <<EOF | istioctl create -f -
```

Comment:
You need to put this into a separate {{< text bash >}} block. Otherwise, the rendering doesn't work properly.